### ISSUE TITLE
Don't work on my computer

### ISSUE THREAD

**Issue by cp287-zz** [eoakley/artifacthelper#14]
Still doesn't work on my computer.
My system: windows 7, two monitors, resolutions: 1920x1080 and 1600x1200. Artifact on first one with resolution 1440x960 16:10 borderless.

Now without any error messages, Only after push the button "Scan Cards" nothing happens.

**Comment by eoakley** [comment]
Can you check if you have a file screen_shot_debugg.png in the installation directory?

Also, please reply with the contents of the file 'Artifact_Helper.launch.pyw.log' in the folder C:\Users\<your_user>\AppData\Roaming

**Comment by cp287-zz** [comment]
Unfortunately no screen_shot at the installation directory, only old one from yesterday.

content of the file  'Artifact_Helper.launch.pyw.log':

...And One For Me, Abaddon, Aghanim's Sanctum, Altar of the Mad Moon, Annihilation, Apotheosis Blade, Arcane Assault, Arcane Censure, Arm the Rebellion, Assassin's Apprentice, Assassin's Shadow, Assassin's Veil, Assured Destruction, At Any Cost, Avernus' Blessing, Axe, Barbed Mail, Beastmaster, Bellow, Better Late Than Never, Bitter Enemies, Blade of the Vigil, Blink Dagger, Bloodseeker, Bolt of Damocles, Book of the Dead, Bounty Hunter, Bracers of Sacrifice, Bristleback, Broadsword, Bronze Legionnaire, Burning Oil, Buying Time, Call the Reserves, Caught Unprepared, Centaur Warrunner, Chainmail, Champion of the Ancient, Cheating Death, Chen, Claszureme Hourglass, Claymore, Cleansing Rite, Clear the Deck, Cloak of Endless Carnage, Collateral Damage, Combat Training, Compel, Conflagration, Coordinated Assault, Corrosive Mist, Crippling Blow, Crystal Maiden, Cunning Plan, Curse of Atrophy, Cursed Satyr, Dark Seer, Defend the Weak, Defensive Bloom, Defensive Stance, Demagicking Maul, Diabolic Revelation, Dimensional Portal, Dirty Deeds, Disciple of Nevermore, Divine Intervention, Divine Purpose, Drow Ranger, Earthshaker, Emissary of the Quorum, Enchantress, Enough Magic!, Escape Route, Fight Through the Pain, Fog of War, Foresight, Forward Charge, Fractured Timeline, Friendly Fire, Fur-lined Mantle, Gank, Glyph of Confusion, Golden Ticket, Grand Melee, Grazing Shot, Hellbear Crippler, Helm of the Dominator, Hero's Cape, Heroic Resolve, Hip Fire, Homefield Advantage, Horn of the Alpha, Howling Mind, Incarnation of Selemene, Intimidation, Iron Branch Protection, Iron Fog Goldmine, Jasper Daggers, Juke, Kanna, 
------
Collected, 100 prices.


Keenfolk Golem, Keenfolk Musket, Keenfolk Plate, Keenfolk Turret, Legion Commander, Legion Standard Bearer, Lich, Lightning Strike, Lion, Lodestone Demolition, Lost in Time, Luna, Lycan, Magnus, Marrowfell Brawler, Mazzie, Meepo, Mercenary Exiles, Messenger Rookery, Mist of Avernus, Murder Plot, Necrophos, New Orders, Nyctasha's Guard, Obliterating Orb, Oglodi Catapult, Oglodi Vandal, Ogre Conscript, Ogre Corpse Tosser, Ogre Magi, Omniknight, Outworld Devourer, Path of the Bold, Path of the Cunning, Path of the Dreamer, Path of the Wise, Payday, Phantom Assassin, Phase Boots, Pick a Fight, Pick Off, Pit Fighter of Quoidge, Platemail, Poaching Knife, Poised to Strike, Prellex, Pugna, Rampaging Hellbear, Ravenhook, Ravenous Mass, Raze, Rebel Decoy, Rebel Instigator, Red Mist Maul, Red Mist Pillager, Relentless Pursuit, Relentless Zombie, Remote Detonation, Rend Armor, Restoration Effort, Revtel Convoy, Revtel Investments, Revtel Signet Ring, Ring of Tarrasque, Rising Anger, Ristul Emblem, Rix, Rolling Storm, Roseleaf Rejuvenator, Routed, Rumusque Blessing, Rumusque Vestments, Satyr Duelist, Satyr Magician, Selemene's Favor, Self Sabotage, Selfish Cleric, Seraphim Shield, Shield of Aquila, Shield of Basilius, Shiva's Guard, Shop Deed, Sister of the Veil, Skywrath Mage, Slay, Smash Their Defenses!, Smeevil Armsmaster, Smeevil Blacksmith, Sniper, Sorla Khan, Soul of Spring, Spot Weakness, Spring the Trap, Stars Align, Steal Strength, Steam Cannon, Stonehall Cloak, Stonehall Elite, Stonehall Pike, Stonehall Plate, 
------
Collected, 200 prices.


Storm Spirit, Strafing Run, Sucker Punch, Sven, Temple of War, The Cover of Night, The Oath, The Omexe Arena, The Tyler Estate, Thunderhide Alpha, Thunderhide Pack, Thunderstorm, Tidehunter, Timbersaw, Time of Triumph, Tinker, Tower Barrage, Treant Protector, Trebuchets, Tresdin's Standards, Troll Soothsayer, Tyler Estate Censor, Unearthed Secrets, Unsupervised Artillery, Untested Grunt, Ursa, Venomancer, Ventriloquy, Vesture of the Tyrant, Vhoul Martyr, Viper, Watchtower, Whispers of Madness, Wingfall Hammer, Winter Wyvern, Wrath of Gold, Zeus, 
------
Collected, 237 prices.


Detecting grid once... (960, 1440, 3)
Detecting grid once... (960, 1440, 3)

**Comment by eoakley** [comment]
Can you replace the contents of the file "pkgs/screen_processor.py" with this one?
[screen_processor.py](https://github.com/eoakley/artifacthelper/blob/master/screen_processor.py)

Then relaunch, and press scan cards a couple of times, and then post the log.

Thanks

**Comment by cp287-zz** [comment]
Thanks a lot for your quick response.
Unfortunately, after replacing the file, the program stopped running. 
Also dont see it in the task manager.

**Comment by eoakley** [comment]
Please post the log again, you might also need to replace the file "pkgs/artifact_helper.py" with this : [artifact_helper.py](https://github.com/eoakley/artifacthelper/blob/master/artifact_helper.py)

**Comment by cp287-zz** [comment]
Ok, replaced arttifact_helper.py.
Log file now:
Traceback (most recent call last):
  File "E:\Users\4Game\AppData\Local\Artifact Helper\Artifact_Helper.launch.pyw", line 31, in <module>
    from artifact_helper import main
  File "E:\Users\4Game\AppData\Local\Artifact Helper\pkgs\artifact_helper.py", line 7
    <!DOCTYPE html>
    ^
SyntaxError: invalid syntax

**Comment by eoakley** [comment]
Just fixed this in the [latest release](https://github.com/eoakley/artifacthelper/releases/latest).

Try the new function "Customize Grid".

If you still have problems, open a new issue with the following:
* Description of your setup (monitor resolutions)
* Full screenshot (all monitors) 
* File **screenshot_debugg.png** from the install dir
* File Artifact_Helper.launch.pyw.log from the folder C:\Users\<your_user>\AppData\Roaming

Thanks for the help!
